src/fal/node_graph.py

```diff
@@ -58,6 +58,32 @@ def _add_after_scripts(
     return graph, nodeLookup
 
 
+def _add_post_hooks(
+    model: DbtModel,
+    upstream_fal_node_unique_id: str,
+    faldbt: FalDbt,
+    graph: nx.DiGraph,
+    nodeLookup: Dict[str, FalFlowNode],
+) -> Tuple[nx.DiGraph, Dict[str, FalFlowNode]]:
+    "Add post-hooks to the graph"
+    post_hooks = (
+        FalScript(faldbt, model, path, True) for path in model.get_post_hook_paths()
+    )
+    post_hook_nodes = (
+        ScriptNode(
+            _script_id_from_path(post_hook.path, model.name, "POST_HOOK"), post_hook
+        )
+        for post_hook in post_hooks
+    )
+    for post_hook_node in post_hook_nodes:
+        graph.add_node(post_hook_node.unique_id)
+        nodeLookup[post_hook_node.unique_id] = post_hook_node
+        # model_fal_node depends on fal_script_node
```

Comment:
Isn't this the exact opposite in reality though? The post-hook depends on the fal node if I understood the implementation correctly.

Reply:
Yeah, you're right